vcd/resource_vcd_org_vdc_with_all_compute_policies_test.go

```diff
@@ -0,0 +1,235 @@
+//go:build vdc || ALL || functional
+// +build vdc ALL functional
+
+package vcd
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccVcdOrgVdcWithAllComputePolicies(t *testing.T) {
+	preTestChecks(t)
+	if !usingSysAdmin() {
+		t.Skip(t.Name() + " requires system admin privileges")
+	}
+
+	var params = StringMap{
+		"VmGroup":                   testConfig.VCD.NsxtProviderVdc.PlacementPolicyVmGroup,
+		"VdcName":                   t.Name(),
+		"OrgName":                   testConfig.VCD.Org,
+		"ProviderVdc":               testConfig.VCD.NsxtProviderVdc.Name,
+		"NetworkPool":               testConfig.VCD.NsxtProviderVdc.NetworkPool,
+		"ProviderVdcStorageProfile": testConfig.VCD.NsxtProviderVdc.StorageProfile,
+		"FuncName":                  t.Name(),
+	}
+	testParamsNotEmpty(t, params)
+
+	resourceName := "vcd_org_vdc." + t.Name()
+	configText := templateFill(testAccCheckVcdVdcAllComputePolicies, params)
+	params["FuncName"] = t.Name() + "-Update"
+	configText2 := templateFill(testAccCheckVcdVdcAllComputePolicies_update, params)
+	debugPrintf("#[DEBUG] CONFIGURATION - creation: %s", configText)
+
+	if vcdShortTest {
+		t.Skip(acceptanceTestsSkipped)
+		return
+	}
+
+	resource.Test(t, resource.TestCase{
+		ProviderFactories: testAccProviders,
+		CheckDestroy:      testAccCheckVdcDestroy,
```

Comment:
Great suggestion!

I took the chance of merging `testAccCheckVmPlacementPolicyDestroyed` and `testAccCheckVmPlacementPolicyDestroyed` into one single function, as the code was doing almost the same and this way we avoid some duplication.

Also the function had a small flaw: if the resource was not present in state, it considered the resource destroyed (but it could be that the searched name was simply misspelled or wrong).

Finally I added your suggestion, now all policies are checked to validate they're destroyed correctly.